draft-ietf-rats-eat.md

```diff
@@ -323,54 +323,50 @@ An entity may have strong security like defenses against hardware invasive attac
 There is no minimum security requirement to be an entity.
 
-## CWT, JWT and Detached EAT Bundle
 
-An EAT is primarily a claims set about an entity based on one of the following:
+## EAT as a Framework
 
-* CBOR Web Token (CWT) {{RFC8392}}
-* JSON Web Token (JWT) {{RFC7519}}
+EAT can be viewed as a framework for defining attestation tokens for specific use cases more so than it is one specific token definition.
+While EAT is based on and compatible with CWT and JWT, it goes further and can be thought of in these terms:
 
-All definitions, requirements, creation and validation procedures, security considerations, IANA registrations and so on from these carry over to EAT.
+* An identification and type system for claims in claims-sets
+* Definitions of common attestation-oriented claims
+* Claims are defined in CDDL and serialized using CBOR or JSON
+* Security envelopes based on COSE and JOSE
+* Nesting of claims sets and tokens to represent complex and compound devices
+* A profile mechanism for specifying and identifying specific token formats for specific use cases
 
-This specification extends those specifications by defining additional claims for attestation.
-This specification also describes the notion of a "profile" that can narrow the definition of an EAT, ensure interoperability and fill in details for specific usage scenarios.
-This specification also adds some considerations for registration of future EAT-related claims.
+EAT uses the name/value pairs the same as CWT and JWT to identify individual claims.
+{{theclaims}} defines common attestation-oriented claims that are added to the CWT and JWT IANA registries.
+As with CWT and JWT, no claims are mandatory and claims not recognized should be ignored.
 
-The identification of a protocol element as an EAT, whether CBOR or JSON encoded, follows the general conventions used by CWT, JWT.
-Largely this depends on the protocol carrying the EAT.
-In some cases it may be by content type (e.g., MIME type).
-In other cases it may be through use of CBOR tags.
-There is no fixed mechanism across all use cases.
+Unlike, but compatible with CWT and JWT, EAT defines claims using Concise Data Definition Language (CDDL) {{RFC8610}}.
+In most cases the same CDDL definition is used for both the CBOR/CWT serialization and the JSON/JWT serialization.
 
-This specification adds one more top-level token type:
+Like CWT and JWT EAT uses COSE and JOSE to provide authenticity, integrity and optionally privacy.
+EAT places new new restrictions on cryptographic algorithms and thus retains all the cryptographic flexibility of CWT, COSE, JWT and JOSE.
+This allows for either symmetric key or public key cryptography and a range of algorithm strengths to accommodate a wide variety of use cases.
 
-* Detached EAT Bundle, {{DEB}}
+EAT goes further than CWT and JWT in allowing nesting of claims sets in several ways to accommodate composite entities that have many subsystems.
```

Comment:
CWT and JWT allow nesting too.  What we want to say here, I think, is that EAT provides pre-canned nesting types (submods and bundles) that allow to describe composite attesters in a natural way.

Reply:
reworded. Thx.